<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveGenerator</class>
 <widget class="QMainWindow" name="CurveGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CurveGenerator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>139</y>
      <width>441</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QSlider" name="uSlider">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>16</width>
      <height>101</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="uLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>50</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>U value: 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="ctrlLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>190</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string># of ctrlpoints: 0</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>246</width>
      <height>112</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="bezier">
       <property name="text">
        <string>Bezier</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="bSplines">
       <property name="text">
        <string>B-Splines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="casteljauSubd">
       <property name="text">
        <string>Subdivision using Casteljau</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="quadricSubd">
       <property name="text">
        <string>Subdivision of Quadric B-Splines</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSlider" name="ctrlSlider">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>21</width>
      <height>121</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QSlider" name="xSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="ySlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>380</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>X Value: -</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>Y Value: -</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="subLabel">
       <property name="text">
        <string>Subdivision #: 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="subdSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>350</y>
      <width>168</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="double_2">
       <property name="text">
        <string>Double</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="triple">
       <property name="text">
        <string>Triple</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCurve_Modeler">
    <property name="title">
     <string>Curve Modeler</string>
    </property>
   </widget>
   <addaction name="menuCurve_Modeler"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
   <slots>
    <signal>mousePress(MouseEvent)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
